doc/whats_new/v0.20.rst

```diff
@@ -176,6 +176,10 @@ Changelog
   precision issues in :class:`preprocessing.StandardScaler` and
   :class:`decomposition.IncrementalPCA` when using float32 datasets.
   :issue:`12338` by :user:`bauks <bauks>`.
+
+- |Fix| Calling :func:`utils.check_array` on pandas `Series`, which
```

Comment:
(I think you can use ``` `pandas.Series` ``` for an intersphinx link)

Reply:
I was wondering after I saw the edit. Would be cool ;)